src/test/java/org/jitsi/srtp/SrtpPerfTest.java

```diff
@@ -84,7 +84,7 @@ public void doEncrypt(int num, int payloadSize)
         }
     }
 
-    public void doPerfTest(int num, int payloadSize)
+    public void doPerfTest(int num, int payloadSize, int numWarmups)
```

Comment:
There are some jvm benchmarking libraries that I think specifically control for warmups and stuff, I wonder if we should investigate them a bit.

Reply:
Worth investigating, though it does look more complex. Are we using anything like that anywhere in Jitsi code currently?

Reply:
No, not that I'm aware.  Don't need to do it as part of this PR but I think this is one of the few places where we're actually measuring absolute performance of one impl vs another, so probably is worth looking at at some point.